taylor-fatima.js

```diff
@@ -0,0 +1,143 @@
+
+//Part 1
+//#1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+
+function calculateCube(num) {
+    console.log(Math.pow(num, 3))
+  }
+  calculateCube(2)
+  
+  //#2
+  //Write a function isAVowel that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise. The vowel could be upper or lower case.
+  
+  function isAVowel(letter) {
+    if (letter === 'a') {
+      return true
+    } else if (letter === 'e'){
+      return true
+    } else if (letter === 'i') {
+      return true
+    } else if (letter === 'o') {
+      return true 
+    } else if (letter === 'u') {
+      return true
+    } else {
+      return false
+    }
+  }
+  console.log(isAVowel('a'))
+  
+  //#3
+  //Write a function getTwoLengths that accepts two parameters (strings). 
+  //The function should return an array of numbers where each number is the length of the corresponding string.
+  
+  function getTwoLengths(str1, str2) {
+   let length = [];
+   length.push(str1.length)
+   length.push(str2.length)
+   return length
+  }
+  console.log(getTwoLengths("hi", "hello"))
+  
+  //#4
+  // Write a function getMultipleLengths that accepts a single parameter as an argument: an array of strings. The function should return an array of numbers where each number is the length of the corresponding string.
+  
+  function getMultipleLengths(arr) {
+    let newArray = [];
+    for (let i = 0; i < arr.length; i++) {
+      newArray.push(arr[i].length) 
+    }
+    return newArray
+  }
+  console.log(getMultipleLengths(["hello", "what", "is", "up", "dude"]))
+  
+  //Part 2
+  //#1
+  //Define a function maxOfThree that takes three numbers as arguments and returns the largest of them. If all numbers are the same, it doesn't matter which one is returned. If the two largest numbers are the same, one of them should be returned.
+  
+  function maxOfThree(num1, num2, num3) {
+    if (num1 > num2 && num1 > num3) {
+      return num1
+    } else if (num2 > num1 && num2 > num3) {
+  return num2 
+   } else if (num3 > num2 && num3 > num1) {
+      return num3
+    } else {
+      return num1
+    }
+  } 
+  console.log(maxOfThree(1, 2, 3))
```

Comment:
your logic here is really close! when I tried `maxOfThree(1, 3, 3)`, I got 1 back, not 3. Try changing a few of your `>` to `>=` to resolve the problem